src/lib/components/core/popups/popups/SerialMonitor.svelte

```diff
@@ -103,7 +103,7 @@ function insertDate() {
     {/if}
     {#if mode === Mode.TEXT}
 	    <div class="content" bind:this={element}>
-	        {#each $log as item (item.id)}
+	        {#each $log as item}
```

Comment:
this is used as a performance optimization by svelte to not require the re-rendering of the entire dom when old items are deleted or items are moved

Reply:
Fixed now